beetsplug/bpd/__init__.py

```diff
@@ -71,7 +71,6 @@
 ITEM_KEYS_WRITABLE = set(MediaFile.fields()).intersection(Item._fields.keys())
 
 # Loggers.
-log = logging.getLogger('beets.bpd')
```

Comment:
This is certainly a special case, but here I think the global logger is actually slightly more intelligible than dynamically hooking up class variables. Maybe we should leave this one how it is?